src/analysis/parents.rs

```diff
@@ -29,3 +29,26 @@ pub fn analyze(env: &Env, type_: &Class, imports: &mut Imports)
     parents.reverse();
     parents
 }
+
+pub fn analyze_interface(env: &Env, type_: &Interface, imports: &mut Imports)
+        -> Vec<StatusedTypeId> {
+    let mut parents = Vec::new();
+
+    for &parent_tid in &type_.prereq_parents {
+        let parent_type = env.type_(parent_tid).to_ref_as::<Class>();
```

Comment:
The prerequisites are usually classes. Not sure if I ever saw an interface depend on other interfaces.